action/images.py

```diff
@@ -122,18 +152,54 @@ def burn_rootfs_image(
         type of the image supported by action native or docker
     """
 
+    os_makedirs("images/rootfs/home")
+
     if image_type == "native":
         get_file(image, "rootfs.tar.xz")
+        image = "rootfs.tar.xz"
     elif image_type == "docker":
-        print("Docker images are not yet supported")
-        sys_exit(1)
+        library, image, tag = docker_image_parse(image)
+
+        print("Preparing your docker image...")
+
+        try:
+            image_proto = DockersaveImage(
+                    image=f"{library}/{image}",
+                    tag=tag,
+                    arch=arch
+            )
+        except StopIteration:
+            print(f"This package is not available in the selected architecture {arch}!")
+            if arch == "riscv64":
+                print("INFO: For riscv64 architecture there are not many official docker images available yet. But very often they are available in the riscv64/distro:tag repository.")
+            exit(1)
+        except requests_HTTPError:
+            print(f"Image {library}/{image}:{tag} not exists!")
+            if tag == "latest":
+                print("INFO: Maybe for this image there are not default 'latest' tag.")
```

Comment:
```suggestion
            print(f"This package is not available for the selected architecture: {arch}!")
            if arch == "riscv64":
                print("INFO: Currently, there are not many official Docker images available for the riscv64 architecture. Alternatives can often be found under 'riscv64/image:tag'.")
            exit(1)
        except requests_HTTPError:
            print(f"Image {library}/{image}:{tag} does not exist!")
            if tag == "latest":
                print("INFO: Perhaps there is no 'latest' tag for this image.")
```
I think it would be nice to print such warnings/errors in color, so that they're easily visible to the user in GitHub logs. There's probably some lib for that. Not in the scope of this PR though.